lib/frr_zmq.h

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Do we need to provide a bit more direction on how to use this new api?

Reply:
Ya, I agree. @eqvinox I can write a few lines to put on top as comments if you want.